lib/grandpa/message.go

```diff
@@ -77,21 +77,57 @@ func (v *VoteMessage) ToConsensusMessage() (*ConsensusMessage, error) {
 	}, nil
 }
 
-// NeighbourMessage represents a network-level neighbour message
-type NeighbourMessage struct {
-	Version byte
-	Round   uint64
-	SetID   uint64
-	Number  uint32
+// VersionedNeighbourPacket represents the enum of nighbor messages
```

Comment:
```suggestion
// VersionedNeighbourPacket represents the enum of neighbour messages
```

also is this an enum?